### PR TITLE
Added support for Consul 0.6.3

### PR DESCRIPTION
**Pull request by xakraz**

### CODE REVIEW

**Comment by kamaradclimber** [comment]
I think we should go back to the upstream cookbook first and then consider the upgrade to 0.6.x

**Comment by xakraz** [comment]
I think that going back to the upstream cookbook will take more effort than this PR.

This PR doesn't mean we have to use it for the upgrade we plan to do in Criteo but it allows the tests defined in chef-repo site-cookbook `consul-server-central` to work.

File: attributes/default.rb [hunk 1]
```diff
@@ -76,10 +82,10 @@
 
 # In the cluster mode, set the default cluster size to 3
 default['consul']['bootstrap_expect'] = 3
-default['consul']['data_dir'] = '/var/lib/consul'
-default['consul']['config_dir'] = '/etc/consul.d'
-default['consul']['logfile'] = '/var/log/consul.log'
-default['consul']['init_style'] = 'init'   # 'init', 'runit', 'systemd'
+default['consul']['config_dir']       = '/etc/consul.d'
```
Comment (kamaradclimber) [inline attributes/default.rb]: don't reorder attributes, it decreases readability of the patch

**Comment by kamaradclimber** [comment]
you should separate linting commits from functional ones

File: attributes/default.rb (hunk 1 above)
Comment (xakraz) [inline attributes/default.rb]: Agree, I should not do multiple changes in the same PR ...

Comment (xakraz) [inline attributes/default.rb]: Done: separated the commits

**Comment by kamaradclimber** [comment]
+1

**Pull request merged**